packages/blade/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,564 @@
+      table: {
+        category: propsCategory.CHIP_GROUP,
+        type: {
+          summary: 'string',
+        },
+      },
+    },
+    defaultValue: {
+      description: 'Sets the initial value of the Chip group',
+      table: {
+        category: propsCategory.CHIP_GROUP,
+        type: {
+          summary: 'string | string[]',
+        },
+      },
+    },
+    size: {
+      description: 'Specifies the size of the rendered Chips withing the ChipGroup',
+      options: ['xsmall', 'small', 'medium', 'large'],
+      control: {
+        type: 'radio',
+      },
+      table: {
+        category: propsCategory.CHIP_GROUP,
+        type: {
+          summary: '"xsmall" | "small" | "medium" | "large"',
+        },
+      },
+    },
+
+    // Chip Props
+    icon: {
+      name: 'icon',
+      description: 'Displays the Blade Icon component within the Chip',
+      type: 'select',
+      options: Object.keys(iconMap),
+      mapping: iconMap,
+      table: {
+        category: propsCategory.CHIP,
+        type: {
+          summary: 'IconComponent',
+        },
+      },
+    },
+
+    // Styled Props
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<ChipGroupProps>;
+
+const ChipTemplate: ComponentStory<typeof ChipGroupComponent> = ({ children, ...args }) => {
+  const chipValues = ['Proprietorship', 'Public', 'Small Business'];
+  return (
+    <Box>
+      <Text marginBottom="spacing.3" marginTop="spacing.3">
+        Select Business type:
+      </Text>
+
+      <ChipGroupComponent {...args}>
+        {chipValues.map((chipValue: string) => (
+          <ChipComponent key={chipValue} value={chipValue} icon={args.icon}>
+            {chipValue}
+          </ChipComponent>
+        ))}
+      </ChipGroupComponent>
+    </Box>
+  );
+};
+
+export const Default = ChipTemplate.bind({});
+Default.storyName = 'Single Selection';
+Default.argTypes = {
+  value: {
+    options: ['Proprietorship', 'Public', 'Small Business'],
+    control: {
+      type: 'select',
+    },
+  },
+  defaultValue: {
+    options: ['Proprietorship', 'Public', 'Small Business'],
+    control: {
+      type: 'select',
+    },
+  },
+};
+
+const MultiSelectChipTemplate: ComponentStory<typeof ChipGroupComponent> = ({
+  children,
+  ...args
+}) => {
+  const chipValues = ['Refunded', 'Failed', 'Pending', 'In Progress'];
+
+  return (
+    <Box>
+      <Text marginBottom="spacing.3" marginTop="spacing.3">
+        Quick Filters:
+      </Text>
+
+      <ChipGroupComponent selectionType="multiple" {...args}>
+        {chipValues.map((chipValue: string) => (
+          <ChipComponent key={chipValue} value={chipValue}>
+            {chipValue}
+          </ChipComponent>
+        ))}
+      </ChipGroupComponent>
+    </Box>
+  );
+};
+
+export const MultiSelection = MultiSelectChipTemplate.bind({});
+MultiSelection.storyName = 'Multi Selection';
+MultiSelection.args = {
+  accessibilityLabel: 'Quick Filters:',
```

Comment:
lets use previous example of Business type itself. Quick filters isn't fitting well here. what you can do is put a dropdown with mutliselect option which will also select the multiple chips

Reply:
@kamleshchandnani Business type is actually `singular` in nature, you can either be Public or Proprietor.

Reply:
so changhe context maybe use what else you're looking wala context

Reply:
Updated.